packages/tools/kadena-cli/src/account/commands/accountFund.ts

```diff
@@ -8,66 +8,62 @@ import { FAUCET_CONSTANTS } from '../../constants/faucet.js';
 import { createCommand } from '../../utils/createCommand.js';
 import { globalOptions } from '../../utils/globalOptions.js';
 
-export const fundCommand: (
-  program: Command,
-  version: string,
-) => void = createCommand(
-  'fund',
-  'fund account',
-  [
-    globalOptions.accountName(),
-    globalOptions.amount(),
-    globalOptions.fungible(),
-    globalOptions.network(),
-    globalOptions.chainId(),
-  ],
-  async (config) => {
-    debug('account-fund:action')({ config });
+export const fundCommand: (program: Command, version: string) => void =
```

Comment:
Please align ur ide :) and remove all the files that are not related to your actual changes from this PR

Reply:
Fixed, removed the non-related files

Reply:
Note: these files were auto formatted using `pnpm format:src` so I am not sure if the problem is in my IDE